ospf6d/ospf6_asbr.c

```diff
@@ -1049,7 +1061,6 @@ void ospf6_asbr_distribute_list_update(int type, struct ospf6 *ospf6)
 	args = XCALLOC(MTYPE_OSPF6_DIST_ARGS, sizeof(void *) * 2);
```

Comment:
As there's only one argument left (pointer to `struct ospf6`), we can pass it directly to `thread_add_timer_msec` and eliminate the allocation.

Reply:
We also don't need to pass type to this function anymore, the debug is always printed only for the first type anyway.

Reply:
Fixed